lib/config/options/index.ts

```diff
@@ -1491,6 +1491,12 @@ const options: RenovateOptions[] = [
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'pruneBranchAfterAutomerge',
+    description: 'Set to true to enable branch pruning after automerging.',
```

Comment:
```suggestion
    description: 'Set to `true` to enable branch pruning after automerging.',
```

@viceice Can we use backticks to get `true` displayed in a monospace font, or do we need to wrap it in a `<code></code>` element?

Reply:
Looking at the existing code, backticks should be enough. Pushed commit for the same.